modules/imagetools/src/main/ts/ephox/imagetools/transformations/ImageResizerWebgl.ts

```diff
@@ -202,6 +202,6 @@ function _loadShader(gl: WebGLRenderingContext, source: string, type: string) {
   return shader;
 }
 
-export default {
+export {
```

Comment:
Interesting... it looks like this whole file is not used anymore :thinking:

Reply:
Logged TINY-4723 to look at this.

Reply:
Ask Josh about it maybe?